src/permissions/models.rs

```diff
@@ -81,6 +82,11 @@ impl PermissionRequest {
                 }
             },
 
+            (GET) (/allpermissions) => {
```

Comment:
The URL should be `/` instead of `/allpermissions` to be more consistent with the rest of the API